tools/getkeys/index.js

```diff
@@ -64,8 +64,7 @@ function stdResponse(err, stdout, stderr) {
     try {
         // Key Vault with restricted access for the FF dev team only
         await getKeys(client, rc, "https://ff-internal-dev-secrets.vault.azure.net/");
-    } catch (e) {
-        console.log("Couldn't get secrets from FF internal keyvault. If you need access make sure you are in the relevant security group.")
-    }
+        console.log("Overriding defaults with values from the FF internal keyvault.")
```

Comment:
From the PR description, it seems this should be in the `catch` block?

Reply:
Most people will not have permissions and hit the catch block, and they should move on silently.  This is fine.  But for folks with the extra permissions, it's nice to mention that you get the different secrets overriding the defaults.